LoopKit/Extensions/FloatingPoint.swift

```diff
@@ -1,13 +1,13 @@
 //
 //  FloatingPoint.swift
```

Comment:
[nit] _Technically_ this is an extension of `Double` not `FloatingPoint`...consider renaming

Reply:
The two below are on FloatingPoint and are sufficiently related that I'm not worried about it.